### ISSUE TITLE
NVDA does not read focused div upon shift+tab

### ISSUE THREAD

**Issue by kaashan** [nvaccess/nvda#10145]
Hi,

I have a requirement where I need a certain `div` (containing important information) to be tab navigable (using tab or shift+tab) so that user can read and understand its  contents. Thus, a tabindex of "0" is assigned to `div`. This `div` also contains some `<a>` links inside it. It is observed that  `div` contents are read when focus moves on it using tab key. On further tab, the `<a>` link is read next. This is expected. However, once the link inside the `div` is focused and we press `shift+tab` key, the `div` content are not read out even though it gains focus.

### Steps to reproduce:
a. Open https://codepen.io/kaashan/pen/KOmGYe in Chrome/Firefox browser
b. Launch NVDA
c. Click on User Name input text field.
d. Press tab key.
e. NVDA reads focused div contents. This is expected.
f. Press tab key again. `<a>` link inside div is focued and NVDA reads link content. This is expected as well.
g. Now, press shift+tab key. `div` content is not read out by NVDA even though it gets focused.

### Actual behavior:
NVDA does not read the `div` contents again when we navigate using `shift+tab` key

### Expected behavior:
NVDA should be able to read the `div` contents again when we navigate using `shift+tab` key

### System configuration
OS: Windows 10 Enterprise
Browser: Chrome/Firefox
Screen reader: NVDA 2019.1.1

#### NVDA installed/portable/running from source:
Installed

#### NVDA version:
NVDA 2019.1.1

#### Windows version:
Windows 10 Enterprise

#### Name and version of other software in use when reproducing the issue:
I checked the scenario with JAWS 2019 and it works well.

#### Does the issue still occur after restarting your PC?
Yes

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
No

**Comment by kaashan** [comment]
Duplicate issue

**Comment by GSW1999** [comment]
Hi. Did u find the soluton for that. Could u please share.
